app/src/main/res/values-es-rES/strings.xml

```diff
@@ -573,1092 +445,795 @@
     <string name="privacy_options_popups_reset">Restablecer la configuración de bloqueo de ventanas emergentes</string>
-
     <!-- This string is displayed on top of the allowed sites list in the allowed pop-ups panel. -->
     <string name="privacy_options_popups_list_header_v1">Los siguientes sitios web tienen permiso para abrir ventanas emergentes:</string>
-
     <!-- This string is displayed in the first line of a text that's displayed in the middle of allowed pop-ups panel instead of the exceptions list
          when there are no exceptions. -->
     <string name="privacy_options_popups_list_empty_first">Cuando abres un sitio web con ventanas emergentes, verás un cuadro de diálogo que te permite bloquear ventanas emergentes.</string>
-
     <!-- This string is displayed in the second line of a text that's displayed in the middle of allowed pop-ups panel instead of the exceptions list
          when there are no exceptions. -->
     <string name="privacy_options_popups_list_empty_second">Una lista de sitios con permisos aparecerá aquí.</string>
-
     <!-- This string is displayed on top of the saved logins list in the Saved Logins panel. -->
     <string name="privacy_options_saved_logins_list_header">Inicios de sesión guardados:</string>
-
     <!-- This string is displayed in the first line of a text that's displayed in the middle of the saved logins panel instead of the logins list
          when there are saved logins. -->
     <string name="privacy_options_saved_logins_list_empty_first">Cuando se pueda guardar un inicio de sesión, verás un cuadro de diálogo que te permite guardarlo.</string>
-
     <!-- This string is displayed in the second line of a text that's displayed in the middle of the saved logins instead of the logins list
          when there are no saved logins. -->
     <string name="privacy_options_saved_logins_list_empty_second">Aquí aparecerá una lista de los inicios de sesión guardados.</string>
-
     <!-- This string labels the Reset button that restores the default saved logins settings values. -->
     <string name="privacy_options_saved_logins_reset">Restablecer los ajustes de inicios de sesión guardados</string>
-
     <!-- This string labels the Clear All button that removes all the saved logins from the stored saved logins. -->
     <string name="privacy_options_saved_logins_clear_all">Borrar todo</string>
-
     <!-- This string labels the Delete button from the Login edit dialog deletes the current login. -->
     <string name="privacy_options_saved_login_delete">Eliminar</string>
-
     <!-- This string is used to label radio buttons for setting ETP in.
          '%1$' Will be replaced at runtime with the Learn More url for ETP. -->
-    <string name="privacy_options_tracking"><![CDATA[Protección mejorada contra el rastreo. (<a href="%1$s">Saber más</a>)]]></string>
-
+    <string name="privacy_options_tracking">Protección mejorada contra el rastreo. (<a href="%1$s">Saber más</a>)</string>
```

Comment:
I hope this string works correctly! In weblate, I received this as a string with modifications, and the commas in the href were missing.

Reply:
Ok, I will build it and test it myself and let you know. Thanks!

Reply:
@jaragunde it turns out this was a problem. Without the CDATA block, the link is removed. The text `Saber mas` is there, but without active link.

I added a new commit putting the CDATA block back. I'm afraid I will have to do this manually for now, for all translations that contain embedded HTML :/

Good catch!